xbmc/platform/android/activity/XBMCApp.h

```diff
@@ -258,7 +259,7 @@ class CXBMCApp
   static void SetRefreshRateCallback(CVariant *rate);
   static void SetDisplayModeCallback(CVariant *mode);
 
-  void RegisterDisplayListener();
+  static void RegisterDisplayListener(CVariant*);
```

Comment:
since this is now a callback it should be renamed to RegisterDisplayListenerCallback

Reply:
I don’t think so. It’s not exactly a callback. It’s just a worker function that shall be executed by the gui thread. It worked exactly like this before my cleanup PR and was  exactly named like it is now. So, I’d like to keep the name.

Reply:
runNativeOnUiThread takes a pointer to a callback function as first argument. And the other 2 functions passed to it are SetRefreshRateCallback and SetDisplayModeCallback

Reply:
since the argument is `(*callback)` and for consistency i‘d vote `RegisterDisplayListenerCallback` indeed. toss a coin? 😂

Reply:
No hard feelings about this. I you feel I should change that can easily do that.